### PR TITLE
Martinas/UI

### PR DESCRIPTION
**Pull request by Mart1nux**
Updated UI for ShowIndividualEvent, ManageIndividualEvent, ManageIndividualVolunteer, ManagePotentialVolunteer.

ShowIndividualEvent:
image

ShowIndividualEvent2:
image

ShowIndividualEvent3:
image

ManageIndividualEvent:
image

ManageIndividualVolunteer:
image

### CODE REVIEW

File: Pages/ManageUserEvents.razor [hunk 1]
```diff
@@ -50,3 +43,10 @@
         }
     }
 </div>
+
+<MatSelect Label="Page size" Value="@perPage" ValueChanged="async(int a) => await Refresh(a)">
+    <MatOption TValue="int" Value="3">3</MatOption>
+    <MatOption TValue="int" Value="5">5</MatOption>
+    <MatOption TValue="int" Value="10">10</MatOption>
+</MatSelect>
+<Pagination TotaPagesQuantity="m_totalPageQuantity" CurrentPage="m_currentPage" Radius="2" SelectedPage="SelectedPage"></Pagination>
```
Comment (cezaris13) [inline Pages/ManageUserEvents.razor:52]: Maybe I just missed but when searching for an event in index page it works perfectly, but on manage volunteer page it doesn't.

File: Pages/Volunteers.razor.cs [hunk 2]
```diff
@@ -61,8 +61,13 @@ public void NavigateToManageVolunteer(Guid id)
         public string FindMostValuableVolunteer()
         {
             IEnumerable<VolunteerAndEquipment> listOfVolunteersWithEquipment = GetPairsOfVolunteersAndEquipment();
-            VolunteerAndEquipment volunteerWithMostEquipment = listOfVolunteersWithEquipment.Aggregate((volunteerWithMostEquipment, nextVolunteer) => nextVolunteer.Size > volunteerWithMostEquipment.Size ? nextVolunteer : volunteerWithMostEquipment);
-            return volunteerWithMostEquipment.VolunteerName + " " + volunteerWithMostEquipment.VolunteerSurname;
+            //
```
[the remark this answers is not in the document]
Comment (Mart1nux) [reply Pages/Volunteers.razor.cs:64]: Removed //.

File: Pages/ManageUserEvents.razor (hunk 1 above)
Comment (Mart1nux) [reply Pages/ManageUserEvents.razor:52]: Yep, my bad, forgot to do the same changes on ManageUserEvents. It should work properly now.

**Review by cezaris13 — APPROVED** [review]
Looks nice, good job

**Pull request merged**